scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -170,25 +168,24 @@ class PostgresStorage(
         }
     }
 
-    override fun readFromStorage(pkg: Package, scannerDetails: ScannerDetails): Result<ScanResultContainer> {
-        val version = Semver(scannerDetails.version)
+    override fun readFromStorage(pkg: Package, scannerCriteria: ScannerCriteria): Result<ScanResultContainer> {
+        // TODO Evaluate the version range
+        val version = scannerCriteria.minScannerVersion
```

Comment:
What about `maxScannerVersion`?

Reply:
This was just a temporary state to make the tests run; hence the TODO comment. In the next commit the query has been reworked to deal with the version range.